stock_picking_batch_extended/views/report_batch_picking.xml

```diff
@@ -13,22 +13,21 @@
             name="report_file"
         >stock_picking_batch_extended.report_batch_picking</field>
     </record>
-
     <template id="report_batch_picking">
         <t t-call="web.html_container">
-            <t t-call="web.external_layout">
-                <div class="page">
-                    <div class="row mt32 mb32">
-                        <div class="col-3">
-                            <strong>Date:</strong>
-                            <span t-esc="now()" t-options="{'widget': 'datetime'}" />
+            <t t-foreach="docs" t-as="doc">
```

Comment:
Don't add this loop here. Do it in another template and passing `t-language` for each document. See the scheme in other standard documents.